### ISSUE TITLE
Missing examples/client

### ISSUE THREAD

**Issue by lsj9383** [ngtcp2/ngtcp2#283]
When I successful build ngtcp2 follow guide, but not found examples/client executable file.

```
[root@~/pkg/ngtcp2(master)]$ tree examples/
examples/
├── client.cc
├── client.h
├── CMakeLists.txt
├── debug.cc
├── debug.h
├── examplestest.cc
├── h09client.cc
├── h09client.h
├── h09server.cc
├── h09server.h
├── http.cc
├── http.h
├── keylog.cc
├── keylog.h
├── Makefile
├── Makefile.am
├── Makefile.in
├── network.h
├── server.cc
├── server.h
├── shared.cc
├── shared.h
├── template.h
├── util.cc
├── util.h
├── util_test.cc
└── util_test.h
```

**Comment by simonasmulevicius** [comment]
I am having the same issue. I suppose that it relates to the unsatisfied requirements. Is there a way to check that all the required packages are correctly installed?

**Comment by simonasmulevicius** [comment]
I found a solution to the missing **client/server** problem. It seems that the `master branch` has some issues with these two example scripts, so I opted for the latest draft branch (currently it is draft-32 https://github.com/ngtcp2/ngtcp2/tree/draft-32).

**Comment by tatsuhiro-t** [comment]
Check the summary of configure script.  Most likely libev is not installed or it cannot find nghttp3 and/or patched OpenSSL.

**Comment by simonasmulevicius** [comment]
Do you mean `libev-dev` when referring to libev?

**Comment by simonasmulevicius** [comment]
Great, it worked - I managed to exchange packets between example server and client using draft-32. My mistake was that I misinterpreted server's message `Using document root ...` as an indication that server halted while in the meantime it was just waiting for new clients to arrive. It would be more convenient for others if server outputed "Waiting for clients..." message.

By the way, can one use draft-32 now or is it missing some substantial functionality?
